qa/workunits/rados/test_envlibrados_for_rocksdb.sh

```diff
@@ -33,7 +33,7 @@ case $(distro_id) in
                 sudo subscription-manager repos --enable "codeready-builder-for-rhel-8-x86_64-rpms"
                 ;;
         esac
-        install git gcc-c++.x86_64 snappy-devel zlib zlib-devel bzip2 bzip2-devel libradospp-devel.x86_64 cmake
+        install git gcc-c++.x86_64 snappy-devel zlib zlib-devel bzip2 bzip2-devel libradospp-devel.x86_64 cmake libarchive
```

Comment:
```suggestion
        install git gcc-c++.x86_64 snappy-devel zlib zlib-devel bzip2 bzip2-devel libradospp-devel.x86_64 cmake libarchive-3.3.3
```

as libarchive could be already installed on the building host, and that was the case in https://tracker.ceph.com/issues/51101. to ensure we have access to a libarchive with zstd support, might want to specify the version as well.

Reply:
sure, done! https://pulpito.ceph.com/nojha-2021-07-12_22:32:46-rados:singleton-master-distro-basic-smithi/